src/hooks/useProposals.js

```diff
@@ -139,6 +141,26 @@ export function useProposal(proposalId, appAddress) {
   return [proposalWithData, blockHasLoaded, loadingProposal]
 }
 
+export function useInactiveProposalsWithStake() {
+  const { account } = useWallet()
+  const { honeypot } = useAppState()
+
+  const supporter = useSupporterSubscription(honeypot, account)
+
+  if (!supporter || !supporter.stakes) {
+    return []
+  }
+  const inactiveStakes = supporter.stakes.filter(stake => {
+    return (
+      stake.proposal.type !== ProposalTypes.Decision &&
```

Comment:
Note that stakes are only part of Proposals and Suggestions so the type check is irrelevant here.